list/internal/list.ts

```diff
@@ -90,4 +90,14 @@ export class List extends LitElement {
   activatePreviousItem(): ListItem | null {
     return this.listController.activatePreviousItem();
   }
+
+  /**
+   * Retrieves the first activated item of the array of items.
+   *
+   * @return A record of the first activated item including the item and the
+   *     index of the item or `null` if none are activated.
+   */
+  get activeItem() {
+    return this.listController.activeItem;
```

Comment:
I don't think we should expose the `{item, index}` object that the controller uses. Instead, I would suggest separate `activeItem` and `activeItemIndex` properties, like [how tabs work](https://github.com/material-components/material-web/blob/a9ee4f5bc1d6702e5dc352eefed13a1d849577e3/tabs/internal/tabs.ts#L45-L99).